module-4/backend/week_four.md

```diff
@@ -5,15 +5,26 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+  I prefer waffle to pivotal
 2. Why do we create staging environments?
+  To keep unfinished products away from production, so that bugs/errors/and precompilation issues can be solved without effecting the live product. 
```

Comment:
you're on the right track - to test your application on an environment that mimics your production environment to ensure bugs/errors/other issues are resolved before making your changes live for users